lib/system.nim

```diff
@@ -1332,6 +1344,7 @@ const
   hasThreadSupport = compileOption("threads") and not defined(nimscript)
   hasSharedHeap = defined(boehmgc) or defined(gogc) # don't share heaps; every thread has its own
   taintMode = compileOption("taintmode")
+  nimEnableCovariance* = defined(nimEnableCovariance) # or true
```

Comment:
So, when I specifically told you that I'm going to do it this way and you agreed, who was I talking to? :P

Anyways, this should be a temporary measure until we decide the fate of covariance, right?